components/infobox/commons/infobox_weapon.lua

```diff
@@ -119,6 +137,43 @@ function Weapon:nameDisplay(args)
 end
 
 function Weapon:setLpdbData(args)
+	local name = args.romanized_name or self.name
+
+	local lpdbData = {
+		name = name,
+		location = self:getStandardLocationValue(args.location),
+		location2 = self:getStandardLocationValue(args.location2),
+		image = args.image,
+		imagedark = args.imagedark or args.imagedarkmode,
```

Comment:
this doesn't exist either

see the following for all available keys in datapoint tables
![B32551D9-00FC-42B4-95E1-175D71942F2B](https://user-images.githubusercontent.com/75081997/177628311-c221e738-4f43-4348-8bd3-aca8a24b320c.jpeg)